keps/77-dynamically-sized-jobs/README.md

```diff
@@ -7,168 +7,281 @@
-  - [Ignore Resize from Kuberay](#ignore-resize-from-kuberay)
 <!-- /toc -->
 
 ## Summary
 
-Enable dynamic sizing of Kueue jobs. For the MVP, we will only focus on supporting RayClusters with autoscaling enabled, but other resources that can benefit from dynamic sizing should be supported eventually.
+Enable dynamic resizing of Kueue-managed workloads by supporting in-place horizontal scaling of job parallelism.
+The primary goal is to allow dynamic adjustment of job parallelism without requiring suspension and requeueing, thereby improving scheduling efficiency and minimizing disruption during resource fluctuations.
 
-See: [Support dynamically sized (elastic) jobs #77](https://github.com/kubernetes-sigs/kueue/issues/77)
+In addition, we aim to extend dynamic resizing support to RayCluster workloads. 
+While the initial focus is on batch/v1.Job and RayCluster, the long-term objective is to generalize support for dynamically sized (elastic) workloads across other Kueue-integrated resource types.
 
 ## Motivation
 
-Kueue currently lacks support for resizing jobs. When a job is resized, Kueue will recreate the Workload representation of the job, leading to a disruptive suspend and requeue process. This limitation hinders the usability of Kueue for resources like RayCluster that sometimes have autoscaling capabilities enabled.
+Kueue currently lacks native support for resizing jobs. Any change in job size leads to the recreation of the associated Workload, resulting in job suspension and requeueing. 
+This disrupts execution and hinders usability for elastic workloads like RayCluster, which rely on in-place autoscaling. 
 
-To properly support RayCluster, Kueue needs to gracefully handle the scale up and scale down of RayCluster nodes. Concretely this means that scaling the resources used by a RayCluster is appropriately reflected in the respective ClusterQueue without needing to suspend the entire RayCluster.
+To support such scenarios, Kueue must gracefully handle horizontal scale-up and scale-down operations without disrupting admitted jobs or re-acquiring quota.
 
 ### Goals
 
-- Gracefully handle resize operations for Kueue jobs (i.e. update quota usage without suspend, enqueue scale ups)
-- Autoscaling RayCluster works with Kueue since it has an autoscaler and there is high demand (MVP)s
-
+- Gracefully handle resize operations for Kueue-managed jobs by updating quota usage without suspending the job or terminating running pods.
+  
 ### Non-Goals
 
-- Vertical scaling of workloads – Kueue will only handle resize operations that scale Pods horizontally 
-- Support resize for other Kueue jobs such as Job, JobSet, etc (future)
-- Resizing of RayJobs
-- Partial Preemption
+- Vertical scaling of workloads – Kueue will only handle resize operations that scale Pods horizontally. 
+- Support resize for other Kueue jobs such as Job, JobSet, etc (future).
+- Resizing of the Workload parent objects (`batchv1/Job`, RayJobs, etc.)
```

Comment:
TBH I'm not sure what is meant in this sentence, the KEP aims to support resizing of parent objects. I see this was taken from the previous version of the KEP. However, I would suggest either clarify the intention or drop the point.

Reply:
My understanding,  it is not a goal of this proposal to orchestrate (initiate, coordinate, etc.) parent Job resizing.

Reply:
That said, I am ok removing this item altogether.

Reply:
> My understanding, it is not a goal of this proposal to orchestrate (initiate, coordinate, etc.) parent Job resizing.

Oh I see, this makes sense. Maybe keep it then as: `Orchestration of resizing of the Job objects`.

Reply:
Updated, PTAL.